KSystemInformer/KSystemInformer.vcxproj

```diff
@@ -33,7 +33,7 @@
     <Configuration>Debug</Configuration>
     <Platform Condition="'$(Platform)' == ''">Win32</Platform>
     <RootNamespace>KSystemInformer</RootNamespace>
-    <WindowsTargetPlatformVersion>$(LatestTargetPlatformVersion)</WindowsTargetPlatformVersion>
```

Comment:
`$(LatestTargetPlatformVersion)`

Reply:
Specific targeting is intentional for now. VS2022 and the newer SDKs don't play nicely with our targeted OSes and architectures.

Reply:
What about at least using more recent WDK 22000 ?
That should still be able to target Windows7 without a problem.

FYI
VS2022 requires WDK 22621 (Win11 22H2) and that does support only Windows10 targets.
It also does support only x64 and arm64 builds.
https://docs.microsoft.com/en-us/windows-hardware/drivers/download-the-wdk

I was wondering what are your plans regarding VS/WDK and `KSystemInformer`.
Do you want to stick with VS2019 and old WDK (like for long time)?
Or would you consider dropping Win7/8 (and x86) support?

Reply:
> What about at least using more recent WDK 22000 ?

That's reasonable, I'll update to use `10.0.22000.0` with the next commit.

> VS2022 requires WDK 22621 

Yep, I'm very familiar with it. Driver development is usually lags behind the latest and greatest anyway. This is pretty standard and Microsoft historically screws up the WDK during transitions like this. I'm hopeful they'll walk back those restrictions a little bit, but we'll see.

> Do you want to stick with VS2019 and old WDK (like for long time)?
> Or would you consider dropping Win7/8 (and x86) support?

As long as we're going to support Win7/8 and x86 we'll stay on the WDK and build tools that Microsoft says supports it. Eventually we'll drop support for those legacy OSes. I don't see that happening immediately. For now I'm interested in getting 3.0 out and people migrated to it with minimal compatibility problems.

I have discussed alternatives with @dmex but I do not want to deal with the struggle of conditionally trying to support something that Microsoft explicitly states is not supported. We recently updated the other projects to have a dependency on the latest SDK and WDK. Which means we can't use `$(LatestTargetPlatformVersion)` for the driver. It is strictly, as of right now, incompatible with what we're targeting.

Reply:
Updated target to `10.0.22000.0`.

Reply:
I'm resolving discussion this for now. I'm happy to extend build support for VS2022 and the newer WDK when the time is appropriate.

A few parting notes:
1. VS2022 and the WDK extension is just broken right now there is a clearly posted regression on the front page of the WDK. Once Microsoft resolves that I will update the build script to provide an optional "latest" switch which chooses VS2022, forces the target to Windows10 and WDK, then omits Win32. This is similar to what @dmex had previously but due to the regression in VS2022 and the WDK extension I want to wait until that gets resolved.
2. Once we deprecate support for Win32 and Win 7/8 I will re-target and update the build script accordingly. Or, in the event Microsoft walks back WDK requirements before then, I will update the WDK version to use the latest.